common/modifiers/00_static_modifiers.txt

```diff
@@ -366,7 +366,7 @@ political_dysfunction = {
 # Babylon RP: Changed value to 10% reduction (Danarca)
 diplomatic_mitigation = {
 	icon = gfx/interface/icons/timed_modifier_icons/modifier_flag_positive.dds
-	country_infamy_decay_mult = 0.10
+	country_infamy_decay_mult = 0.20
```

Comment:
Another big change, what's the reasoning behind this? With the scaling infamy we solved a big problem already :s

Reply:
This change was done pre-infamy modifiers, as far as I know, and now that we have actual penalties for Infamy we need proper decay that the game provides or else people are going to struggle to do what even historical nations did. I think 20 could be too high, but at least 15 is reasonable considering you have to actively not have any deals with other nations to keep that modifier at 100% effectiveness.